### PR TITLE
gh-91 Include environment info in the About tab

### PR DESCRIPTION
**Pull request by ghillert**
* Add link to details page on About page
* Add ability to copy environment info to clipboard

Resolves https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/91
Resolves https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/160

### CODE REVIEW

File: ui/app/scripts/shared/views/about.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
 <h2>Get the Spring Cloud Data Flow Shell</h2>
 
 <p class="index-page--subtitle">As an alternative to the Dashboard UI, you can also
-   <a href="https://repo.spring.io/libs-snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/{{dataflowVersionInfo.app.version}}/spring-cloud-dataflow-shell-{{dataflowVersionInfo.app.version}}.jar"
+   <a href="https://repo.spring.io/libs-snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/{{dataflowVersionInfo.versionInfo.core.version}}/{{dataflowVersionInfo.versionInfo.core.version}}.jar"
       target="_blank">download the compatible version of the Shell</a> ({{dataflowVersionInfo.app.version}}).</p>
```
Comment (trisberg) [inline ui/app/scripts/shared/views/about.html]: shouldn't this line also use `dataflowVersionInfo.versionInfo.core.version`?

I can change this on merge

File: ui/app/scripts/shared/views/about.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
 <h2>Get the Spring Cloud Data Flow Shell</h2>
 
 <p class="index-page--subtitle">As an alternative to the Dashboard UI, you can also
-   <a href="https://repo.spring.io/libs-snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/{{dataflowVersionInfo.app.version}}/spring-cloud-dataflow-shell-{{dataflowVersionInfo.app.version}}.jar"
+   <a href="https://repo.spring.io/libs-snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/{{dataflowVersionInfo.versionInfo.core.version}}/{{dataflowVersionInfo.versionInfo.core.version}}.jar"
```
Comment (trisberg) [inline ui/app/scripts/shared/views/about.html]: looks like we lost the  `spring-cloud-dataflow-shell-` part of the link, will re-add on merge

**Comment by trisberg** [comment]
I don't see any of the `platformSpecificInfo` shown on the detail page although I get them if I use the "Copy Details to Clipboard" button

Also, we have the heading "Platform-specific Information" twice, should we add "- Deployer" and "-Task Launcher" to make it clearer?

**Comment by ghillert** [comment]
@trisberg Updated PR. This should address all issues.

**Comment by trisberg** [comment]
LGTM, squashed and merged as commit e9c48e2dce6824ef3270a82dd3a503acf04e0e0b

**Pull request closed without merging**